cylc/flow/scheduler.py

```diff
@@ -1159,23 +1150,17 @@ def _load_workflow_params(self, row_idx, row):
             LOG.info('LOADING workflow parameters')
         key, value = row
         if key in self.workflow_db_mgr.KEY_INITIAL_CYCLE_POINT_COMPATS:
-            if self.is_restart and self.options.icp == 'reload':
-                LOG.debug(f"- initial point = {value} (ignored)")
-            elif self.options.icp is None:
-                self.options.icp = value
-                LOG.info(f"+ initial point = {value}")
+            self.options.icp = value
+            LOG.info(f"+ initial point = {value}")
         elif key in self.workflow_db_mgr.KEY_START_CYCLE_POINT_COMPATS:
```

Comment:
Since #4313, initial and start cycle points cannot change on restart. But still need to set `self.options.icp`/`startcp` as they are used later to set `self.config.initial_point`/`start_point`.